README.md

```diff
@@ -442,29 +442,82 @@ module.exports = function(app) {
   });
 };
 ```
+#### Migrating models with GeoPoint types before loopback-connector-mysql@5.x
```

Comment:
Suggestion: Making the title more prominent because it is a breaking change.  Perhaps something like "Breaking changes with GeoPoint since 5.x "